coinstacks/bitcoin/api/src/models.ts

```diff
@@ -78,14 +78,14 @@ export type BtcTxSpecific = {
 /**
  * Contains info about an unspent transaction output
  */
-export interface UtxoResponse {
+export interface Utxo {
```

Comment:
We can update this type to be our own shape that we want unchained to respond with. Where I have been starting with this strategy is only passing back what we are actually using in the chain adapter side, and then maybe making the `pubkey` vernacular abstraction if we want instead of separating `address` and `path`. This will allow us to leverage blockbook, but build out our own API how we see fit and try not to just copy blockbook because we happen to be using blockbook

Reply:
are you asking for changes here? the attributes I see there make sense for a utxo at a glance. even though chain-adapters doesn't use `coinbase` for example that's a standard enough thing to include up front IMO

Reply:
With all of the other types, I have been starting slim with what we as currently the only client are using and more can easily be added if there is need for the information by external clients. We can include all of it, but the point I am trying to drive in as we build out this API is that we should be thinking about our own unchained abstraction and not using blockbook types verbatim just because they exist and are the easy crutch to use. I want to separate our API abstraction from blockbook's abstraction.

Reply:
Utxo is pretty generic so they might end up being very similar, I just want us to be cognizant of the intention of trying to build out our own types for our unchained API abstraction.